### PR TITLE
Update ref hardware specs

### PR DESCRIPTION
**Pull request by ggwpez**
Changes:
- Updating the reference hardware ram requirement to 32 GB. Formerly this was set to 64 GB and then reduced to 16 GB without much thought. Now after some (internal) discussions we want to raise it again.
- The mentioned `c2d-highcpu-8` instances dont offer NVMe drives therefore changing to `n2`. @oleg-plakida PTAL

### CODE REVIEW

**Review by DrW3RK — APPROVED** [review]

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md
Comment (mateo-moon) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]: We don't actually run benchmarks in CI on AWS instances. Also i think the right type for AWS instance would be `c6i.4xlarge` because `c6id.2xlarge` has only 16GB if it's absolutely mandatory to keep memory the same. Unfortunately there is no one to one identical instances between gcp and aws.

**Comment by DrW3RK** [comment]
Hey @ggwpez ! Should I wait for @oleg-plakida 's response before I merge this?

**Comment by ggwpez** [comment]
> Hey @ggwpez ! Should I wait for @oleg-plakida 's response before I merge this?

Yes please. He setup the hardware. Just want to be sure that this is the correct ID for the VM type.

**Review by mateo-moon — APPROVED** [review]

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md
Comment (ggwpez) [reply docs/maintain/maintain-guides-how-to-validate-polkadot.md]: Okay thanks. I updated it.

**Comment by ggwpez** [comment]
Should be good now @DrW3RK

**Pull request merged**
